### ISSUE TITLE
Suggest giving a description what to copy for the “Copy” button in the Preview Changes dialog 

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#8769]
## Details about Problem
NuGet version: NuGet Client Dev\5.5.0.6284
VS Version: Master\29429.48
OS: Windows-10-Enterprise-19H1-04-05-2019
## Detailed repro steps 
1. Create a template via VS.
2. Open PM UI of the project by right-clicking the project->Manage NuGet Packages… in Solution Explorer.
3. Search a package in Browse tab and try to install it.
## Expected
There should be a description what to copy (like the description for OK button or a tooltip) for the Copy button in the prompt Preview Changes dialog.
## Actual
There is no description to tell user what to copy for the Copy button in the prompt Preview Changes dialog as below screenshot.
![NoDescriptionForCopy](https://user-images.githubusercontent.com/20632625/67843315-b1b8e100-fb36-11e9-803f-0f703c68b85a.png)

**Comment by v-luzh** [comment]
It still reproes on VS Main\35221.199 + NuGet Client Dev\6.12.0.81.

**Comment by v-luzh** [comment]
It still reproes on VS Main\12206.68+NuGet Client Dev\7.12.0.4. So, I will reopen it.  Suggest calling this button 'Copy Changes'.
